openmetadata-ui/src/main/resources/ui/src/components/Explore/ColumnSummary/ColumnSummary.component.tsx

```diff
@@ -0,0 +1,67 @@
+import { Divider, Space, Typography } from 'antd';
```

Comment:
License header is missing for the new file

Reply:
Done